src/daemon/processor/animations.rs

```diff
@@ -99,9 +103,6 @@ impl Transition {
             2 => self.right(new_img, outputs, sender, stop_recv),
             3 => self.top(new_img, outputs, sender, stop_recv),
             4 => self.bottom(new_img, outputs, sender, stop_recv),
```

Comment:
We are missing 'grow' and 'outer' transitions in here. For 'grow' we can simply call 'any'. For 'outer' we might be ok with leaving it out for now. Nevertheless, **remember to update the random value that comes before this**. Currently it is `match r % 8`, which will generate numbers in the range `[0-7]`, but we only have numbers in the range `[0-4]` in this PR.

Reply:
how about the `any` transition being both grow and outer randomly?

Reply:
That sounds pretty good!